package.json

```diff
@@ -36,17 +36,19 @@
   },
   "homepage": "https://github.com/hotgazpacho/apollo-server-cache-dynamodb#readme",
   "dependencies": {
-    "@apollo/utils.keyvaluecache": "^1.0.2"
+    "apollo-server-caching": "^3.3.0"
```

Comment:
I believe that `apollo-server-caching` has been deprecated in favor of `@apollo/utils.keyvaluecache`. We should be using the new package instead.